cmd/earthly/main.go

```diff
@@ -1176,6 +1176,82 @@ Set up a whole custom git repository for a server called example.com, using a si
 				},
 			},
 		},
+		{
+			Name: "preview",
+			Subcommands: []*cli.Command{
+				{
+					Name:        "project",
+					Description: "Create and manage Earthly projects",
+					UsageText:   "earthly project (ls|rm|create|member)",
+					Flags: []cli.Flag{
+						&cli.StringFlag{
+							Name:     "org",
+							EnvVars:  []string{"EARTHLY_ORG"},
+							Usage:    "The name of the organization to which the project belongs. Required when user is a member of multiple.",
+							Required: false,
+						},
+					},
+					Subcommands: []*cli.Command{
+						{
+							Name:        "ls",
+							Usage:       "List all projects that belong to the specified organization",
+							Description: "List all projects that belong to the specified organization",
+							UsageText:   "earthly project [--org <organization-name>] ls",
+							Action:      app.actionProjectList,
+						},
+						{
+							Name:        "rm",
+							Usage:       "Remove an existing project from the organization",
+							Description: "Remove an existing project from the organization",
+							UsageText:   "earthly project [--org <organization-name>] rm <project-name>",
+							Action:      app.actionProjectRemove,
+						},
+						{
+							Name:        "create",
+							Usage:       "Create a new project in the specified organization",
+							Description: "Create a new project in the specified organization",
+							UsageText:   "earthly project [--org <organization-name>] create <project-name>",
+							Action:      app.actionProjectCreate,
+						},
+						{
+							Name:        "member",
+							Description: "Manage project members",
+							UsageText:   "earthly project member (ls|rm|add|update)",
+							Subcommands: []*cli.Command{
+								{
+									Name:        "add",
+									Usage:       "Add a new member to the specified project",
+									Description: "Add a new member to the specified project",
+									UsageText:   "earthly project [--org <organization-name>] member add <project-name> <user-id-or-email> <permission>",
+									Action:      app.actionProjectMemberAdd,
+								},
```

Comment:
When not providing the right arguments, it would be nice to say what arguments are expected, rather than just the number of them.

<img width="899" alt="Screen Shot 2022-07-06 at 2 33 18 PM" src="https://user-images.githubusercontent.com/446771/177647388-fbdaee7c-ba23-4529-a304-780dab81c32c.png">

Reply:
Sure, sounds good.